django_ipam/static/django-ipam/js/hosts-infinite-scrolling.js

```diff
@@ -0,0 +1,87 @@
+/*jslint sloppy: true, browser:true */
+/*globals onUpdate*/
+
+function initHostsInfiniteScroll($, current_subnet, address_add_url) {
+    "use strict";
+    var renderedPages = 5;
+    var fetchedPages = [];
+    var busy = false;
+    var nextPageUrl = "/api/v1/subnet/" + current_subnet + "/hosts";
+    var lastRenderedPage = 0; //1 based indexing (0 -> no page rendered)
```

Comment:
you could also combine the multiple declaration with only one var statement:

```javascript
var renderedPages = 5,
    fetchedPages = [],
    // etc...
    lastRenderedPage = 0;
```

Reply:
Done, didn't know about such syntax :smile: